docs/reference/circuit.rst

```diff
@@ -0,0 +1,23 @@
+Circuits
+========
+
+.. automodule:: dwave.gate.circuit
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+mixedproperty module
+--------------------
```

Comment:
Valid point. This is straightforward to fix, but I'm trying to think what would be best long-term since I've added module descriptions to the module `__init__.py` files, which won't be rendered if I remove the top-level `.. automodule::`. I'm guessing either

* Keep the module descriptions in the `__init__.py` and have the top-level `.. automodule::`, but add another heading for it (e.g., Circuit Module, for the example above) if it contains any methods, which most of them shouldn't.
* Move the module description to the rst files instead of having them in the  `__init__.py`.

Maybe I'm missing something here though, and there's a better way.